test/Makefile

```diff
@@ -9,7 +9,7 @@ frida_version := 9.1.1
 test_sources := $(wildcard re/frida/*.java)
 test_classes := $(patsubst %.java,%.class,$(test_sources))
 
-android_jar := $(shell echo $(ANDROID_SDK_ROOT))/platforms/android-23/android.jar
+android_jar := $(ANDROID_SDK_ROOT)/platforms/android-23/android.jar
```

Comment:
The `echo` hack was to expand the tilde in the default `ANDROID_SDK_ROOT`. Perhaps we should do: `$(abspath $(ANDROID_SDK_ROOT)` instead?